tests/conftest.py

```diff
@@ -3120,10 +3137,27 @@ def finalizer():
         Delete the cloned PVCs
 
         """
+        pv_objs = []
+
+        # Get PV form PVC instances and delete PVCs
         for instance in instances:
             if not instance.is_deleted:
+                pv_objs.append(instance.backed_pv_obj)
                 instance.delete()
                 instance.ocp.wait_for_delete(instance.name)
 
+        # Wait for PVs to delete
```

Comment:
this code is a dup of https://github.com/red-hat-storage/ocs-ci/pull/3215/files#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R2961-R2973

Please create a function for it

Reply:
Created helpers.wait_for_pv_delete function to handle this.

Reply:
thanks!